controller/space.go

```diff
@@ -108,12 +108,11 @@ func (c *SpaceController) Create(ctx *app.CreateSpaceContext) error {
 		}
 
 		// Similar to above, we create a root iteration for this new space
-		userActive := false
 		newIteration := iteration.Iteration{
 			ID:         uuid.NewV4(),
 			SpaceID:    rSpace.ID,
 			Name:       rSpace.Name,
-			UserActive: &userActive,
+			UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c